cloudformation-templates/zs_cc_cf_template_simple.yaml

```diff
@@ -231,21 +245,132 @@ Resources:
     Type: 'AWS::EC2::SecurityGroup'
     Properties:
       VpcId: !Ref VPCID
-      GroupDescription: Enable intra cloud comm
-      SecurityGroupIngress:
-        - IpProtocol: TCP
-          FromPort: '22'
-          ToPort: '22'
+      GroupName: cc-mgmt-sg
+      GroupDescription: Security group for Cloud Connector management interface
+  MgmtEgressRule1:
+      Type: AWS::EC2::SecurityGroupEgress
+      Properties:
+          Description: "Required: CC outbound TCP/443"
+          GroupId: !Ref MgmtSecurityGroup
+          IpProtocol: "tcp"
+          FromPort: 443
+          ToPort: 443
+          CidrIp: 0.0.0.0/0
+  MgmtEgressRule2:
+      Type: AWS::EC2::SecurityGroupEgress
+      Properties:
+          Description: "Required: CC Mgmt outbound UDP/123"
+          GroupId: !Ref MgmtSecurityGroup
+          IpProtocol: "udp"
+          FromPort: 123
+          ToPort: 123
+          CidrIp: 0.0.0.0/0
+  MgmtEgressRule3:
+      Condition: CreateZsSupportRule
+      Type: AWS::EC2::SecurityGroupEgress
+      Properties:
+          Description: "Recommended: CC Mgmt outbound Zscaler Remote Support TCP/12002"
+          GroupId: !Ref MgmtSecurityGroup
+          IpProtocol: "tcp"
+          FromPort: 12002
+          ToPort: 12002
+          CidrIp: 199.168.148.101/32
+  MgmtIngressRule1:
+      Type: AWS::EC2::SecurityGroupIngress
+      Properties:
+          Description: "Recommended: SSH to CC management"
+          GroupId: !Ref MgmtSecurityGroup
+          IpProtocol: "tcp"
+          FromPort: 22
+          ToPort: 22
           CidrIp: 0.0.0.0/0
+      
   ServiceSecurityGroup:
     Type: 'AWS::EC2::SecurityGroup'
     Properties:
       VpcId: !Ref VPCID
-      GroupDescription: Enable intra cloud comm
-      SecurityGroupIngress:
-        - IpProtocol: '-1'
-          FromPort: '0'
-          ToPort: '65535'
+      GroupName: cc-svc-sg
+      GroupDescription: Security group for Cloud Connector service interfaces
+  ServiceEgressRule1:
```

Comment:
Can we add better rule names please. Something more descriptive than ServiceEgressRule1

Reply:
@anichari names fixed in https://github.com/zscaler/cloud-native-aws-cloud-connector-deploy/pull/44/commits/d5133a38a05a821c1784e49e3cb596c1cbb772f8